### PR TITLE
Step three towards moving timer changes.

### PR DESCRIPTION
**Pull request by EricWF**
This patch is the third and final step before merging the timer changes.

This patch contains 3 main changes along with some small misc changes:
1. Create string_util.h and string_util.cc with string utility from benchmark.cc
2. Cleanup includes.
3. Update copyright.

### CODE REVIEW

File: src/string_util.h [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#ifndef BENCHMARK_STRING_UTIL_H_
+#define BENCHMARK_STRING_UTIL_H_
+
+#include <string>
+#include <sstream>
+#include <utility>
+
+namespace benchmark {
+
+void ToExponentAndMantissa(double val, double thresh, int precision,
```
Comment (dominichamon) [inline src/string_util.h]: which of these actually need to be in the header? i'm guessing not all.

would you mind checking before you merge?

Comment (EricWF) [inline src/string_util.h]: Done.

**Comment by dominichamon** [comment]
LGTM modulo tests passing.

**Comment by EricWF** [comment]
Tests are failing again due to an existing bug. pushing.

**Pull request merged**